packages/backend/tsconfig.json

```diff
@@ -40,7 +40,7 @@
     // "resolvePackageJsonExports": true,                /* Use the package.json 'exports' field when resolving package imports. */
     // "resolvePackageJsonImports": true,                /* Use the package.json 'imports' field when resolving imports. */
     // "customConditions": [],                           /* Conditions to set in addition to the resolver-specific defaults when resolving imports. */
-    // "resolveJsonModule": true,                        /* Enable importing .json files. */
+    "resolveJsonModule": true /* Enable importing .json files. */,
```

Comment:
Nit: alignment of the comment seems off now

Reply:
Prettier seems to do that with all of the other uncommented config lines. I think Ill leave this as is.